components/Accordion/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+
+import './style.scss';
+
+const Accordion = ({ children, title }) => {
+  const [isActive, setIsActive] = useState(false);
+
+  const handleToggle = () => {
+    setIsActive(!isActive);
+  }
+  
+  return (
+    <div className="Accordion">
```

Comment:
This should probably be marked up as an actual `<details>` element, or at least set [the appropriate `[aria-role]` for the wrapper and summary sections](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-details)! CC: @trishaprile